eng/pipelines/templates/stages/platform-matrix.json

```
@@ -9,7 +9,7 @@
       "macos-11": { "OSVmImage": "macos-11", "Pool": "Azure Pipelines" }
     },
     "JavaTestVersion": [ "1.8", "1.17" ],
-    "AZURE_TEST_HTTP_CLIENTS": [ "okhttp", "netty" ],
+    "AZURE_TEST_HTTP_CLIENTS": [ "jdk-httpclient", "vertx", "okhttp", "netty" ],
```

```suggestion
    "AZURE_TEST_HTTP_CLIENTS": [ "vertx", "okhttp", "netty" ],
```

`jdk-httpclient` should be removed from this list as the matrix may be created with `azure-core-http-jdk-httpclient` being used in a Java 8 test run where it will fail, given the JDK 12+ requirement.